extras/onewire/README.md

```diff
@@ -0,0 +1,7 @@
+# Yet another one wire driver for the ESP8266
+
+This is a port of bit banging one wire driver based on nodemcu implementaion.
+
+For all aspect regarding license, please check in their code.
```

Comment:
One last thing, the license/copyright notices mentioned here seem to be missing?

Some (all?) of the library code appears to be from pjrc's OneWire library, so we should definitely acknowledge upstream author and license that in the comments at the top of each file - as well as any other relevant copyrights and licensing.